specifyweb/frontend/js_src/lib/components/Merging/index.tsx

```diff
@@ -171,6 +183,16 @@ function Merging({
     true
   );
 
+  // const [mergeId, setMergeId] = React.useState<string | undefined>(undefined);
+
+  const [aborted, setAborted] = React.useState<boolean | 'failed' | 'pending'>(
+    false
+  );
+
+  function handleAbort() {
```

Comment:
don't need to create a function for this.
just create an inline function in the component:
```
onAbort={()=>setMergeId(undefined)}
```

Reply:
I intended to call the abort API in the function handleAbort that's why I created that function